IBM/carbon-addons-iot-react
# fix(valuecard): data state tooltip opens behind other cards

Closes #1040

**Summary**
GRAPHITE team has implemented the data-state-error value cards in Reference application Dashboard. Since it has multiple tiles/cards on the page. here if we are clicking on the no data or error icon the tooltip is opening behind other cards. As per discussion with team, they are mentioning that they are not directly using the IOT component and not modifying tooltip in GRAPHITE hence it seems to be fixed in IOT.

![image](https://user-images.githubusercontent.com/5167091/77566892-9de79000-6ec6-11ea-85fc-c9a98828026b.png)


**Change List (commits, features, bugs, etc)**
-Removed the tooltip wrapper which had the 'data-floating-menu-container' attribute. This wrapper was originally added as a fix recommended by carbon to handle the offset when the tooltip container was scrolling and there was a parent with position:fixed. That scenario is considered a corner case and non of the tooltips in our components handle that.
-Modified a card in the story `dashboard -> only value cards` to show the data-state so that the issue and fix could be easily tested

The issue #1040 also requests a fix for better tooltip edge positioning which I believe is outside the scope of this fix. See discussion in the original issue.

**Acceptance Test (how to verify the PR)**
1. Go to the story 'dashboard -> only value cards' in the build for this issue. (https://deploy-preview-1041--carbon-addons-iot-react.netlify.com/?path=/story/watson-iot-dashboard--only-value-cards)
2. Click the data state info icon and make sure it is not hidden by another card
